### PR TITLE
I did task 6-10

### PR DESCRIPTION
**Pull request by aklutskaya**

### CODE REVIEW

File: src/homework/day8/ListTasks/TaskSix.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        List<String> birds = Arrays.asList("Чайка", "Дрозд", "Бусел", "Голубь", "Воробей", "Цапля");
+
+        for (String bird : birds) {
+            System.out.println("--" + bird + "--");
+        }
+        System.out.println();
+
+        int count = 0;
+        Pattern pattern = Pattern.compile("[аоуея]");
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/ListTasks/TaskSix.java]: I think that we have more Vowels in RU

File: src/homework/day8/ListTasks/TaskSeven.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        elements.add("Breadcrumb");
+
+        for (String element : elements) {
+            System.out.print(element + " ");
+        }
+        System.out.println();
+
+        int count = 0;
+        for (String element : elements) {
+            if (element.contains("-") || element.contains(" ")) {
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/ListTasks/TaskSeven.java]: I think this approach will fail if String hava space in the beggining or in the end

Comment (IrynaAlesiyuk) [reply src/homework/day8/ListTasks/TaskSeven.java]: And "-" in the beggining and end

File: src/homework/day8/ListTasks/TaskNine.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        System.out.println("Произведение всех чисел: " + count);
+
+        double sum = 0;
+        for (double doubleElement : doubles) {
+            sum += doubleElement - (int) doubleElement;
+        }
+        System.out.println("Сумма всех дробных частей: " + sum);
+
+        for (int i = 0; i < doubles.size(); i++) {
+            System.out.print(doubles.get(i) + " ");
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/ListTasks/TaskNine.java]: это вернет скорее dooble, а нужно только целую часть

**Review by IrynaAlesiyuk — CHANGES_REQUESTED** [review]

File: src/homework/day8/ListTasks/TaskSix.java (hunk 1 above)
Comment (aklutskaya) [reply src/homework/day8/ListTasks/TaskSix.java]: Added more vowels

File: src/homework/day8/ListTasks/TaskNine.java (hunk 3 above)
Comment (aklutskaya) [reply src/homework/day8/ListTasks/TaskNine.java]: fixed

File: src/homework/day8/ListTasks/TaskSeven.java (hunk 2 above)
Comment (aklutskaya) [reply src/homework/day8/ListTasks/TaskSeven.java]: fixed

**Review by IrynaAlesiyuk — APPROVED** [review]

**Pull request merged**